insights/specs/default.py

```diff
@@ -418,6 +418,16 @@ def semid(broker):
     ls_etc = simple_command("/bin/ls -lanR /etc")
     ls_lib_firmware = simple_command("/bin/ls -lanR /lib/firmware")
     ls_ocp_cni_openshift_sdn = simple_command("/bin/ls -l /var/lib/cni/networks/openshift-sdn")
+    ls_origin_local_volumes_pods_base = listdir("/var/lib/origin/openshift.local.volumes/pods")
+
+    @datasource(ls_origin_local_volumes_pods_base)
```

Comment:
I think we have discussed it yesterday. This datasource is not needed. Use the `listdir` directly.